src/main/java/at/hannibal2/skyhanni/features/garden/GardenNextJacobContest.kt

```diff
@@ -508,8 +509,9 @@ object GardenNextJacobContest {
         }
     }
 
-    private fun isEnabled() = LorenzUtils.inSkyBlock && config.display
-        && (GardenAPI.inGarden() || config.everywhere)
+    private fun isEnabled() = ((OutsideSbFeature.NEXT_JACOB_CONTEXT.isSelected() && !LorenzUtils.inSkyBlock) ||
+        (LorenzUtils.inSkyBlock && (GardenAPI.inGarden() || config.showOutsideGarden)))
+        && config.display
```

Comment:
maybe something more like this idk


    private fun isEnabled() = config.display && (OutsideSbFeature.NEXT_JACOB_CONTEXT.isSelected() ||
    GardenAPI.inGarden() || (LorenzUtils.inSkyBlock && config.showOutsideGarden))

Reply:
No that would cause show outside sb to also make it show outside garden (e.g. in crimson isles)